### PR TITLE
TCH model of Triadic Closure & Homophily without Preferential Attachment

### PR DESCRIPTION
**Pull request by mannbach**
A simple growth model of Triadic Closure and Homophily, but without Preferential Attachment.

### CODE REVIEW

File: netin/generators/tch.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        -------
+        tuple
+            homophily values of the graph (majority, minority)
+        """
+        h_MM = None
+        h_mm = None
+        return h_MM, h_mm
+
+    def infer_triadic_closure(self) -> float:
+        """
```
Comment (lisette-espin) [inline netin/generators/tch.py:164]: add a @TODO or a message that implementation is not yet done.

File: netin/generators/tch.py [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    def info_computed(self):
+        """
+        Shows the (computed) properties of the graph.
+        """
+        Homophily.info_computed(self)
+        TriadicClosure.info_computed(self)
+
+    def infer_homophily_values(self) -> Tuple[float, float]:
+        """
+        Infers analytically the homophily values of the graph.
```
Comment (lisette-espin) [inline netin/generators/tch.py:152]: add a @TODO or a message that implementation is not yet done.

**Review by lisette-espin — APPROVED** [review]
@mannbach please add a @TODO comment on the inference of homophily and triadic closure indicating that these two methods still need to be implemented. Other than that, this is great, thanks!

**Comment by mannbach** [comment]
Would it make sense to throw an error (e.g., [NotImplementedError](https://docs.python.org/3/library/exceptions.html#NotImplementedError)) or a warning? Or would it crash somewhere unexpectedly?

**Pull request merged**